pages.it/common/git-gc.md

```diff
@@ -0,0 +1,24 @@
+# git gc
+
+> Ottimizza il repository locale ripulendolo dai file non necessari.
+> Maggiori informazioni: <https://git-scm.com/docs/git-gc>.
+
+- Ottimizza il repository:
+
+`git gc`
+
+- Ottimizza il repository in modo più aggressivo, impiegando più tempo:
+
+`git gc --aggressive`
+
+- Non eliminare gli oggetti non tracciati (sono eliminati di default):
+
+`git gc --no-prune`
+
+- Non mostrare alcun output:
+
+`git gc --quiet`
+
+- Mostra utilizzo completo:
```

Comment:
I kept this because present also in the original. However, I think that it's redundant, as it applies to any other git command

Reply:
Yeah, probably right but should be solved in another PR. If we don't have more valuable examples it's fine to keep it.